javascript/code-challenges/queueWithStacks/queue-with-stacks.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+
+const Stack = require('route');
+
+class PuesdoQueue {
+  constructor(value) {
+    this.stackOne = new Stack();
+    this.stackTwo = new Stack();
+    this.front = null;
+    this.rear = null;
+  }
+
+  enqueue(value) {
+    let currentNode = new Node(value);
+    currentNode = this.front;
+    if (this.rear) { this.rear.next = currentNode; }
+    this.rear = currentNode;
+  }
+
+}
+
```

Comment:
You look like you were off to a great start! If you have any questions drop them in the comments here or on Canvas!